.cirrus.yml

```diff
@@ -75,9 +75,37 @@ testing_task:
         failure_journal_log_script: '${_JOURNALCMD} || true'
 
 
+# This task runs `make vendor` followed by ./hack/tree_status.sh to check
+# whether the git tree is clean.  The reasoning for that is to make sure
+# that the vendor.conf, the code and the vendored packages in ./vendor are
+# in sync at all times.
+vendor_task:
+
+    only_if: $CIRRUS_CHANGE_MESSAGE !=~ '.*\*\*\*\s*CIRRUS:\s*TEST\s*IMAGES\s*\*\*\*.*'
+
+    env:
+        CIRRUS_WORKING_DIR: "/var/tmp/go/src/github.com/containers/buildah"
+        GOPATH: "/go"
+        GOSRC: "/go/src/github.com/containers/buildah"
```

Comment:
It would be good for these to all align. Esp. `$CIRRUS_WORKING_DIR` since that's where the code gets checked out.  I recommend we use `/var/tmp/go` as `${GOPATH}` because it matches what we do in podman and makes doing rootless testing (in the future) MUCH easier.  Also if we ever need to test in atomic or silverblue, messing with any top-level subdirs of `/` is a huge "no-no".  So this would be preferred:

```yaml
env:
￼         GOPATH: "/var/tmp/go"
￼         GOSRC: "${GOPATH}/src/github.com/containers/buildah"
         CIRRUS_WORKING_DIR: "${GOSRC}"
```

Reply:
Note: the original path mismatch is an artifact of podman's `quay.io/libpod/gate:latest` (which has a special entrypoint script).  Here, for `docker.io/library/golang:1.13` it can be simplified as above.